src/docs/terraform-authentication.md

```diff
@@ -0,0 +1,13 @@
+# Authentication for Terraform
```

Comment:
Looks like we both solved this problem, my mistake. This doc exists already in `/bicep` from #377 and can be removed from this PR.

Reply:
Deleted!